package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Command-line tool to visually diff JavaScript components",
   "scripts": {
     "eslint": "eslint --ext .js,.jsx",
-    "test": "npm run --silent eslint -- .",
+    "test": "jest",
```

Comment:
I don't think I understand how that is set up. Is this done automagically by travis? I don't see it referenced in `.travis.yml`. 

I'll put it back.